pkg/security/resolvers/tags/resolver.go

```diff
@@ -93,12 +96,12 @@ func (t *DefaultResolver) Stop() error {
 // NewResolver returns a new tags resolver
 func NewResolver(config *config.Config, telemetry telemetry.Component) Resolver {
 	if config.RemoteTaggerEnabled {
-		options, err := remote.NodeAgentOptionsForSecurityResolvers()
+		options, err := remote.NodeAgentOptionsForSecurityResolvers(rootconfig.Datadog())
 		if err != nil {
 			log.Errorf("unable to configure the remote tagger: %s", err)
 		} else {
 			return &DefaultResolver{
-				tagger: remote.NewTagger(options, taggerTelemetry.NewStore(telemetry)),
+				tagger: remote.NewTagger(options, rootconfig.Datadog(), taggerTelemetry.NewStore(telemetry)),
```

Comment:
I am not sure about how I am passing the config component here. 

Any suggestions?

Reply:
I think using it like that is fine for now, we are working on cleaning up this tagger usage to use the tagger component directly which would remove the need to pass the config component on its own